src/Tests/dotnet.Tests/CommandTests/ToolUpdateLocalCommandTests.cs

```diff
@@ -311,6 +311,29 @@ public void GivenFeedVersionIsLowerRunPackageIdItShouldThrow()
                 _manifestFilePath));
         }
 
+        [Fact]
+        public void GivenFeedVersionIsLowerWithDowngradeFlagRunPackageIdItShouldSucceeds()
```

Comment:
In this case, will it downgrade the package?  I think it probably shouldn't.

It looks like the scenario is when the latest version of the package on the feed is lower than the version currently installed.  In that case, I think the update command should succeed but not actually change the version installed.  This should be the case whether or not the `--allow-downgrade` option is specified.

Reply:
Thank you! I added a `--version` flag into the test and it should refer to the package with the specified lowered version.

Reply:
Great.  Are we sure the behavior where you don't specify the version flag is that it won't downgrade?